### ISSUE TITLE
Pagination styles

### ISSUE THREAD

**Issue by rdmtinez** [coleifer/sqlite-web#150]
The pagination:
```
<nav> <ul class="pagination">...</ul> </nav>
```
could use a better color scheme, it was very hard to spot and I thought I was missing data for bit.

**Comment by coleifer** [comment]
Yeah you're right, it looks busted out of the box. I'm using bootswatch "Yeti" theme and the 4.x version seems to have this issue. I've sorta worked around it for the time being.
